frappe/__init__.py

```diff
@@ -316,6 +317,9 @@ def throw(msg, exc=ValidationError, title=None):
 
 	:param msg: Message.
 	:param exc: Exception class. Default `frappe.ValidationError`"""
+	msg = str(jinja2.escape(msg))
```

Comment:
not a good idea, this will also escape tracebacks... maybe just strip `<script>` in js?

Reply:
You're right! On it!